testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -141,7 +143,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
      * Change this whenever you add or remove subprojects for distribution-packaged plugins (lib/plugins).
      */
     int getPackagedPluginsJarCount() {
-        81
+        80
```

Comment:
Do we know what went away here?

Reply:
Diagnostics was a plugin jar.  Now base- is a core lib and software- is an extension, so -1.